qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -393,6 +393,21 @@ def test_get_without_target_name(self):
             # if the int() fails then that means there's a valid stat output
             pass
 
+    def test_get_subdir_creation(self):
```

Comment:
I think it will be nice have a clearer name for this test (like `test_get_doesnt_create_dir`). Or at least add a docstring for this test method.  What about something like `"""Test that "get" command doesn't create directories when the path it receives contains directory names besides the targeted file"""`?

Reply:
You can use [write_file()](https://github.com/ceph/ceph/blob/master/qa/tasks/cephfs/mount.py#L655) instead if you wish to. It can be used here  like this `self.mount_a.write_file(<args>)`

Reply:
Okay

Reply:
Added and pushed,  Thanks!